builder/builder.mk

```diff
@@ -172,7 +185,65 @@ gotest: test-ready
 		-it $(shell $(BUILDER)) go build -o /dev/null ./cmd/edgectl
 .PHONY: gotest
 
-test: gotest pytest
+# Ingress v1 conformance tests, using KIND and the Ingress Conformance Tests suite.
+ingresstest:
+	@printf "$(CYN)==> $(GRN)Running $(BLU)Ingress v1$(GRN) tests from $(INGRESS_CONFORMANCE_TESTS_DIR)$(END)\n"
+	@[ -n "$(DEV_REGISTRY)" ] || { printf "$(RED)ERROR: no DEV_REGISTRY defined$(END)\n"; exit 1; }
+	@[ -n "$(AMB_IMAGE)" ] || { printf "$(RED)ERROR: no AMB_IMAGE defined$(END)\n"; exit 1; }
+	@[ -n "$(INGRESS_CONFORMANCE_TESTS_DIR)" ] ||  { printf "$(RED)ERROR: no INGRESS_CONFORMANCE_TESTS_DIR defined$(END)\n"; exit 1; }
+	@[ -d "$(INGRESS_CONFORMANCE_TESTS_DIR)" ] ||  { printf "$(RED)ERROR: $(INGRESS_CONFORMANCE_TESTS_DIR) does not seem a valid directory$(END)\n"; exit 1; }
+
+	@# make sure the `ingress-controller-conformance` executable has been built
+	@[ -x $(INGRESS_CONFORMANCE_TESTS_DIR)/ingress-controller-conformance ] || make -C $(INGRESS_CONFORMANCE_TESTS_DIR)
+
+	@printf "$(CYN)==> $(GRN)Creating cluster with KIND$(END)\n"
+	kind delete cluster 2>/dev/null || /bin/true
+	kind create cluster --image $(KIND_IMAGE)
+
+	@printf "$(CYN)==> $(GRN)Showing some cluster info:$(END)\n"
+	@kubectl cluster-info
+	@kubectl version
+
+	@printf "$(CYN)==> $(GRN)Loading Ambassador (from the Ingress conformance tests) with image=$(AMB_IMAGE)$(END)\n"
+	@for f in $(INGRESS_CONFORMANCE_TESTS_DIR)/examples/ambassador/*.yaml ; do \
```

Comment:
Instead of relying on yaml samples from the ingress-conformance project (which we don't fully manage and could be a pain to update), can we use the manifests from ambassador.git/templates/ambassador?

Reply:
Working on that.